person.rb

```diff
@@ -24,6 +28,11 @@ def correct_name
     @name
   end
 
+  def class1=(class1)
+    @class1 = class1
+    class1.students.push(self) unless class1.students.include?(self)
+  end
+
```

Comment:
- [ ] According to the many-to-many relationship (A person has many rentals), a method for adding rentals should be present in the Person class. Check out the example in a previous lesson: [many-to-many relationship](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more information.